docs/development.md

```diff
@@ -109,7 +152,7 @@ This instruction set applies for both dev server and production server, with sli
 ### As administrator
 
 1. Go to any administrator page, e.g `/web/admin/home`. You may be prompted to log in.
-   You will be granted access only if your account has admin permission as defined in `build.properties`.
+   You will be granted access only if your account has admin permission as defined in `build.properties`. To log in using your admin name you have to edit `build.properties` to grant your username permission and run datastore emulator as defined above. 
```

Comment:
If we are to make it more straightforward, some example might work even better:)

Reply:
This is not the right place to do it:
- As mentioned in the issue thread, it is not the logging in action itself that requires Datastore/emulator.
- The need for Datastore/emulator applies whether you want to be recognized (notice that I didn't say "log in") as student or instructor or admin.